wis2node/util.py

```diff
@@ -113,3 +131,15 @@ class EnvVarLoader(yaml.SafeLoader):
     EnvVarLoader.add_constructor('!path', path_constructor)
 
     return yaml.load(fh, Loader=EnvVarLoader)
+
+
+def yaml_write(fh: str, content: dict) -> None:
```

Comment:
Rename to `yaml_dump` (in the same way we drop in `wis2node.util.yaml_load` for `yaml.load`).